js/tooltip.js

```diff
@@ -471,7 +471,7 @@
     var that = this
     clearTimeout(this.timeout)
     this.hide(function () {
-      that.$element.off('.' + that.type).removeData('bs.' + that.type)
+      that.$element.off('mouseenter focus blur mouseleave', '.' + that.type).removeData('bs.' + that.type)
```

Comment:
`that.type` is not intended to be a selector. Presumably you want something like:
```js
.off('mouseenter.' + that.type + ' focus.' + that.type + ' blur.' + that.type + ' mouseleave.' + that.type)
```